src/context/FFContext.tsx

```diff
@@ -51,6 +53,7 @@ export const FFContextProvider: FC<FFContextProviderProps> = ({
 }) => {
   const [loading, setLoading] = useState<boolean>(true)
   const [flags, setFlags] = useState<FFContextValue['flags']>({})
+  const [clientInstance, setClient] = useState<FFContextValue['client']>()
```

Comment:
really no big deal but should it be `setClientInstance`?

Reply:
Good eye 👍🏻